internal/manifests/manifestutils/probes.go

```diff
@@ -0,0 +1,20 @@
+package manifestutils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// TempoReadinessProbe returns a readiness Probe spec for tempo components.
+func TempoReadinessProbe() *corev1.Probe {
+	return &corev1.Probe{
+		ProbeHandler: corev1.ProbeHandler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: TempoReadinessPath,
+				Port: intstr.FromString(HttpPortName),
+			},
+		},
+		InitialDelaySeconds: 15,
```

Comment:
are those fine hard coded? or should be allow some configuration here?

Reply:
I think it's fine hard coded, these values mean that kubernetes waits for 15 seconds until it starts checking the readiness of the pod, and uses a timeout of 1 second for the readiness probe.

In case these values make issues later we can make them configurable, but I'm a bit reluctant to add config options for every tiny knob. wdyt?

Reply:
It is fine to me. I think we can proceed in this way